changelog/5025.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added dynamic z axis option to `sunpy.visualization.animator`.
```

Comment:
Is this a z axis?

Reply:
Was confused about that actually, when I asked about it in the #4718 this is what @dstansby stated so I was under the assumption that it's correct.

Reply:
This sets the image min and max limits on a 2D image. I don't think that is a Z axis.

Reply:
Yeah, so I guess the PR as a whole was pointing to a separate issue being mentioned under #4718

Reply:
No this does address that issue.

Reply:
So, should I change the name of this PR to something else?

Reply:
The name of the PR doesn't really matter.